src/component/Header.js

```diff
@@ -14,9 +15,9 @@ function Header (props) {
             <nav>
                 <h1>place-my-order.com</h1>
                 <ul>
-                    <li className={pathname === "/" ? "active" : ""}><NavLink to="/">Home</NavLink></li>
-                    <li className={pathname === "/restaurants" ? "active" : ""}><NavLink to="/restaurants">Restaurants</NavLink></li>
-                    <li className={pathname === "/order-history" ? "active" : ""}><NavLink to="/order-history">Order History</NavLink></li>
+                    <li className={pathname === "/" ? "active" : ""}><Link to="/">Home</Link></li>
```

Comment:
Isn't there some sort of React Router built in way of adding this `active` class, I just feel like there is, and `pathname ===` and `pathname.startWith()` feel clunky.

I could be wrong, maybe this is just the way to do it.

Reply:
I think RR can do this, but only to the link, not to the container.

```js
<Route
  path={to}
  children={({ match }) => (
    <li className={match ? "active" : ""}>
      <Link to={to} />
    </li>
  )}
/>
```

Reply:
Cleaned this up so the Header is a Route component and gets an appropriate match for equality comparisons, but as Christopher noted the class needs to be set on the <a> container which can't be done with builtin react-router methods.